example/ida/utils.py

```diff
@@ -132,7 +132,7 @@ def __init__(self, expr):
         self.expr = expr
 
         # Initial translation
-        text = Translator.to_language(self.languages[0]).from_expr(self.expr)
+        text = Translator.to_language(self.languages[0])().from_expr(self.expr)
```

Comment:
I'm not quite sure of how to handle that, but to_language(...) might return an already instanciated Translator, to avoid the weird double-() syntax. One of the good things of the factory pattern is to have no idea of how the object is instanciated. But the current syntax allows to pass arguments to the translator instance easily.

Maybe something like `Translator.to_language(cls, target_lang, *args, **kwargs)` could do the job? Not quite sure.